R/fct_SQLite.R

```diff
@@ -369,3 +496,72 @@ db_get_review <- function(
       dplyr::as_tibble()
   })
 }
+
+update_db_version <- function(db_path, version = "1.1") {
```

Comment:
Is it possible to create a test for this to verify if the result is as expected? It is a bit difficult for me to see if conversion is exactly as intended.

Reply:
@jthompson-arcus as discussed on Teams today: we will look into how to test a database update with the function `update_db_version` in a future PR